ktor-client/ktor-client-core/common/src/io/ktor/client/call/HttpClientCall.kt

```diff
@@ -142,8 +116,16 @@ data class HttpEngineCall(val request: HttpRequest, val response: HttpResponse)
  * Constructs a [HttpClientCall] from this [HttpClient] and with the specified [HttpRequestBuilder]
  * configured inside the [block].
  */
-suspend fun HttpClient.call(block: suspend HttpRequestBuilder.() -> Unit = {}): HttpClientCall =
-    execute(HttpRequestBuilder().apply { block() })
+@Deprecated(
+    "Unbound [HttpClientCall] is deprecated. Consider using [HttpResponse] in request method instead.",
```

Comment:
I guess the message should be about using `request()` instead of `call()`, not about `HttpResponse`. Please, check here and below.